### PR TITLE
Grpc query api

### PR DESCRIPTION
**Pull request by devopkeith**
# {grpcQueryAPI}

- [ ] You've read the [Contributor Guide](https://github.com/recro/medulla/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/recro/medulla/blob/main/CODE_OF_CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

## Summary of the changes

{Summary} (Less than 80 characters)

## Description

{Detail}

Fixes #{issue number}

### CODE REVIEW

File: grpc-API.proto [hunk 1]
```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+option csharp_namespace = "grpcAPI.Protos";
+
+service QueryAPI {
+
+    rpc GetTermItem (GetTermItemRequest) returns (TermItem);
```
Comment (keithlogan94) [inline grpc-API.proto:7]: Rename to GetRecord

File: grpc-API.proto [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    rpc GetAllTermItems (GetAllTermItemsRequest) returns (stream TermItems);
```
Comment (keithlogan94) [inline grpc-API.proto:8]: GetRecords

File: grpc-API.proto [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    rpc AddTermItem (AddTermItemRequest) returns (TermItem);
```
Comment (keithlogan94) [inline grpc-API.proto:10]: AddRecord

File: grpc-API.proto [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    rpc UpdateTermItem (UpdateTermItemRequest) returns (TermItem);
```
Comment (keithlogan94) [inline grpc-API.proto:11]: UpdateRecord

File: grpc-API.proto [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    rpc DeleteTermItem (DeleteTermItemRequest) returns (DeleteTermItem);
```
Comment (keithlogan94) [inline grpc-API.proto:12]: DeleteRecord

File: grpc-API.proto [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+    rpc InsertTermItem (stream TermItem) returns (InsertTermItem);
```
Comment (keithlogan94) [inline grpc-API.proto:14]: You don't need both an Insert and an Add, they do the same thing. Remove this one

File: grpc-API.proto [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    rpc Test (google.protobuf.Empty) returns (google.protobuf.Empty);
```
Comment (keithlogan94) [inline grpc-API.proto:15]: Can be removed

**Comment by AndrewBabbitt97** [comment]
Closing due to no open related issue.

**Pull request closed without merging**